code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/ProjectResourceManager.java

```diff
@@ -203,19 +142,27 @@ protected Object deleteInternal(String type, String id, final Object obj, ApiReq
         if (!(obj instanceof Account)) {
             return new Object();
```

Comment:
Seems like maybe returning null or even throwing an exception is a better idea than returning a new Object()